sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/distortion_corrector/distortion_corrector.hpp

```diff
@@ -43,17 +43,29 @@
 #include <deque>
 #include <memory>
 #include <string>
+#include <tuple>
 
 namespace autoware::pointcloud_preprocessor
 {
 
+struct AngleConversion
+{
+  // Equation for the conversion between sensor azimuth coordinates and Cartesian coordinates:
+  // sensor azimuth coordinates = offset_rad + sign * cartesian coordinates;
+  // offset_rad is restricted to be a multiple of 90, and sign is restricted to be 1 or -1.
+  float offset_rad{0};
+  float sign{1};
+  float offset_rad_threshold{0.087f};  // (5 / 180) * M_PI
+  float sign_threshold{0.1f};
+};
```

Comment:
These can be mate `static const` or `static constexpr` so they will still be semantically part of `AngleConversion` but they are not mutable, and they don't show up in the constructor.